docs/tutorials/working_with_large_catalogs.ipynb

```diff
@@ -2,22 +2,395 @@
  "cells": [
```

Comment:
<p><code>catalog.plot_pixels</code> exists now right? can we use that everywhere? And can we explain that plot_pixels is plotting the hipscat partitions that are in the catalog.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/236/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>